include/alpaka/rand/distribution/NormalReal.hpp

```diff
@@ -0,0 +1,119 @@
+/* Copyright 2025 Mehmet Yusufoglu, Tim Hanel
+ * SPDX-License-Identifier: MPL-2.0
+ */
+#pragma once
+
+#include "alpaka/math.hpp"
+#include "alpaka/rand/concepts.hpp"
+#include "alpaka/rand/distribution/UniformReal.hpp"
+
+#include <cmath>
+
+namespace alpaka::rand::distribution
+{
+    namespace internal
+    {
+        /**
+         * @brief Scalar Box–Muller transform for floating-point types.
+         *
+         * Generates independent standard normal deviates from a uniform
+         * scalar engine. The implementation caches the second sample so
+         * that a pair of uniforms yields two normal values.
```

Comment:
```suggestion
         * Generates independent standard normal samples from a uniform
         * scalar engine. The implementation caches the second sample so
         * that a pair of uniforms yields two normal values.
```
Did you mean samples here?